documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -40,6 +40,30 @@ spec:
     # ...
 ----
 
+[[cors-configuration]]
+== Cross-Origin Resource Sharing configuration
+
+By default, Cross-Origin Resource Sharing (CORS) for Cruise Control is disabled and can only be enabled for read-only access.
+This means that applications running in origins different from Strimzi components cannot send POST requests to the Cruise Control API. 
+However, these same applications can still send GET requests to the Cruise Control API when cors is enabled for Cruise Control.
+This prevents third-party applications from manipulating Kafka cluster state through Cruise Control without Strimzi approval but still allows third-party applications to access read-only cluster information from Cruise Control.
+
```

Comment:
_Add a sub-heading and a new sentence:_

.Enabling CORS for Cruise Control

You enable CORS in `Kafka.spec.cruiseControl.config`.